includes/wikia/api/WikisApiController.class.php

```diff
@@ -296,6 +296,10 @@ public function getWikisUnderDomain() {
 			}
 		}
 
+		if ( !WikiFactory::isPublic( $cityId ) ) {
+			throw new ResourceGoneException();
```

Comment:
What would happen in cases where the English wiki at the root was closed, but language path wikis exist (and so we serve the index page at the root, and should have robots rules for the language versions of the wiki)?